apps/edxapp/templates/services/nginx/route_cms.yml.j2

```diff
@@ -25,4 +24,4 @@ spec:
     targetPort: "{{ edxapp_nginx_cms_port }}-tcp"
   to:
     kind: Service
-    name: "edxapp-nginx-{{ deployment_stamp }}"
+    name: "edxapp-nginx-{{ prefix }}"
```

Comment:
more like a suffix?

[edit] after reading the rest of the code I think it should actually be called `stack`.

Reply:
I agree with you on the semantic, I used `prefix` because it's the current variable name on the master branch.

There is 60+ occurrences of this term in 25 files, I'm afraid it will complicate the review of this PR if I rename it here. Maybe I can address this in another PR, what do you think ?

Reply:
yes of course!

Reply:
https://github.com/openfun/arnold/issues/601